packages/engine/Source/Core/createWorldTerrain.js

```diff
@@ -31,12 +32,27 @@ import IonResource from "./IonResource.js";
  *
  */
 function createWorldTerrain(options) {
+  deprecationWarning(
+    "createWorldTerrain",
+    "createWorldTerrain was deprecated in CesiumJS 1.102.  It will be removed in 1.104.  Use createWorldTerrainAsync instead."
+  );
+
   options = defaultValue(options, defaultValue.EMPTY_OBJECT);
 
-  return new CesiumTerrainProvider({
-    url: IonResource.fromAssetId(1),
+  const provider = new CesiumTerrainProvider({
     requestVertexNormals: defaultValue(options.requestVertexNormals, false),
     requestWaterMask: defaultValue(options.requestWaterMask, false),
   });
+
+  provider._readyPromise = CesiumTerrainProvider._initializeReadyPromise(
```

Comment:
I know this is temporary code, but since it's not obvious and "looks bad" you might want to add a comment.

Or on the other hand, why does this code need to be changed at all? Can't it still just call the old constructor as-is since that's deprecated but still works?

Reply:
The old constructor will work, but it will throw an additional deprecation error for a part of the API that the user did not actually use. Happy to go the simpler route and use the old constructor if we're not worried about the second unrelated deprecation error. Otherwise, I will add a comment to explain.